lib/shared/addon/components/cluster-driver/driver-linodelke/component.js

```diff
@@ -0,0 +1,360 @@
+      const newTag = get(this, 'newTag');
+
+      if (newTag) {
+        tags.pushObject(newTag);
+        set(this, 'cluster.lkeEngineConfig.tags', tags);
+        set(this, 'newTag', '');
+      }
+    },
+    deleteTag(idx) {
+      const tags = get(this, 'cluster.lkeEngineConfig.tags') || [];
+
+      set(this, 'cluster.lkeEngineConfig.tags', tags.filter((tag, index) => index !== idx));
+    },
+
+    // for node pools
+    addSelectedNodePool() {
+      const selectedNodePoolObj = get(this, 'selectedNodePoolObj');
+      const selectedNodePoolList = get(this, 'selectedNodePoolList');
+
+      if (selectedNodePoolObj.id) {
+        // add to list
+        selectedNodePoolList.pushObject(selectedNodePoolObj);
+
+        // clear selected
+        set(this, 'selectedNodePoolType', '');
+        set(this, 'selectedNodePoolObj', {});
+      }
+    },
+    deleteNodePool(id) {
+      const selectedNodePoolList = get(this, 'selectedNodePoolList');
+
+      set(this, 'selectedNodePoolList', selectedNodePoolList.filter((n) => n.id !== id))
+    }
+  },
+
+
+  // For languages
+  languageChanged: observer('intl.locale', function() {
+    const lang = get(this, 'intl.locale');
+
+    if (lang) {
+      this.loadLanguage(lang[0]);
+    }
+  }),
+  clusterNameChanged: observer('cluster.name', function() {
+    const clusterName = get(this, 'cluster.name');
+
+    set(this, 'cluster.lkeEngineConfig.name', clusterName);
+    set(this, 'cluster.lkeEngineConfig.label', clusterName);
+  }),
+  clusterDescriptionChanged: observer('cluster.description', function() {
+    const clusterDescription = get(this, 'cluster.description');
+
+    set(this, 'cluster.lkeEngineConfig.description', clusterDescription);
+  }),
+
+  setSelectedNodePoolObj: observer('selectedNodePoolType', async function() {
+    const nodePoolTypes = await get(this, 'nodeTypes');
+    const selectedNodePoolType = get(this, 'selectedNodePoolType');
+
+    if (selectedNodePoolType) {
+      const ans = nodePoolTypes.find((np) => np.id === selectedNodePoolType);
+
+      set(this, 'selectedNodePoolObj', {
+        ...ans,
+        count:    1,
+        memoryGb: ans.memory / 1024,
+        diskGb:   ans.disk / 1024
+      });
+    } else {
+      set(this, 'selectedNodePoolObj', {});
+    }
+  }),
+  setNodePools: observer('selectedNodePoolList.@each.count', function() {
+    const selectedNodePoolList = get(this, 'selectedNodePoolList');
+    const nodePools = selectedNodePoolList.map((np) => {
+      return `${ np.id }=${ np.count }`
+    })
+
+    set(this, 'cluster.lkeEngineConfig.nodePools', nodePools);
+  }),
+
+  // to prefil selected node pool list for edit mode
+  prefillSelectedNodePoolListObserver: observer('nodeTypes.[]', function() {
+    this.prefillSelectedNodePoolList();
+  }),
+
+  // Any computed properties or custom logic can go here
+
+  // For Cluster Configuration Step
+  clusterConfigTitle: computed('intl.locale', 'langChanged', function() {
```

Comment:
I could be missing something but similar to my above comment about the observer, I am not sure why you would need these? You should be able to use the translation in the hbs file directly without the need to setup computed watchers. We do this everywhere in the ember UI so I suspect this may just be a left over artifact from the port. Please use the translation directly in the hbs for this, `clusterConfigDetail`, `accessConfigTitle`, and `accessConfigDetail` below.